go/libkb/chain_link.go

```diff
@@ -836,20 +837,9 @@ func (c *ChainLink) Unpack(m MetaContext, trusted bool, selfUID keybase1.UID, pa
 			return err
 		}
 		payload = []byte(sdata)
-
-		if tmp.sigVersion == KeybaseSignatureV1 {
-			// check that payload_json matches payload in sig
-			sigPayload, err := tmp.Payload()
-			if err != nil {
-				return err
-			}
-
-			payloadFixed := c.fixPayload(payload, c.id)
-
-			if !FastByteArrayEq(payloadFixed, sigPayload) {
```

Comment:
Since we're just checking that the `payload_json` matches the json blob in the signature, how about just use the json blob in the signature and forgo the check. This saves bandwidth. I'm pretty sure this is safe, but please double-check my work here.